### ISSUE TITLE
Custom Colors

### ISSUE THREAD

**Issue by apdavid** [sqjin/CellChat#293]
is there a way to change the colors that are assigned to the cell types? 

We have a color palette that we are using from the UMAP, and we tried using the color palette that you used in your package, but for some clusters there isn't enough contrast, so we would like to modify the default colors of CellChat if that is possible.

**Comment by happylearning423** [comment]
@apdavid Yes, I think you can set the colors when plotting. Please check the parameter of each function.

**Comment by sqjin** [comment]
@apdavid Yes, I think you can set the colors when plotting. Please check the parameter of each function.
